podaac/controllers/default_controller.py

```diff
@@ -0,0 +1,527 @@
+import connexion
+import six
+
+from podaac import util
+
+
+import logging
+import os
+import sys
+import time
+import boto3
+import pymysql
+import pandas as pd
+import csv
+import json
+from datetime import datetime
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+DB_PASSWORD_SSM_NAME=''
+DB_PASSWORD=''
+DB_HOST='127.0.0.1'
+DB_NAME='test'
+DB_USERNAME='root'
+DB_PORT = 3306
+
+#DB_HOST = os.environ['DB_HOST']
+#DB_NAME = os.environ['DB_NAME']
+#DB_USERNAME = os.environ['DB_USERNAME']
+#DB_PORT = 3306
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def format_json(cur, feature_id, exact, time):
+    """
+
+    Parameters
+    ----------
+    cur
+    swot_id
+    exact
+    time
+
+    Returns
+    -------
+
+    """
+    # Fetch all results from mySQL query
+    results = cur.fetchall()
+
+    data = {}
+
+    if results is None:
+        data['error'] = f"404: Results with the specified Feature ID {feature_id} were not found."
+    elif len(results) > 5750000:
+        data['error'] = f'413: Query exceeds 6MB with {int(len(results.split(",")))} hits.'
+
+    else:
+        data['status'] = "200 OK"
+        data['time'] = str(time) + " ms."
+        #data['search on'] = {"featureID": feature_id}
+        data['type'] = "FeatureCollection"
+        data['features'] = []
+        i = 0
+        print(len(results))
+        for t in results:
+            if ((t['time'] != '-999999999999')): #and (t['width'] != '-999999999999')):
+                feature = {}
```

Comment:
Would using the geojson library make this easier? https://pypi.org/project/geojson/